airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/RelationalDbQueryUtils.java

```diff
@@ -18,7 +18,9 @@
 public class RelationalDbQueryUtils {
 
   public static String getIdentifierWithQuoting(final String identifier, final String quoteString) {
-    return quoteString + identifier + quoteString;
+    return (identifier.startsWith(quoteString) && identifier.endsWith(quoteString))
```

Comment:
Can you add a comment here explaining this is to handle double-quoted values? 
Also found that 
if (condition) {
} else {
}

is more readable. WDYT

Reply:
> Can you add a comment here explaining this is to handle double-quoted values? Also found that if (condition) { } else { }
> 
> is more readable. WDYT

added a comment and replaced with if-else